### PR TITLE
Draft an about page

### PR DESCRIPTION
**Pull request by wbamberg**
Draft of an About page. I have copied much of this content from our project repo: https://github.com/openwebdocs/project - I'm not sure if it would be better to have less in here and point more to there, or to move stuff from the project repo into here, or whether some degree of duplication is OK.

Anyway, this is a start. Happy to get feedback!

### CODE REVIEW

**Comment by Elchi3** [comment]
https://inspiring-snyder-9401af.netlify.app/about is 404. How do I preview this?

> I'm not sure if it would be better to have less in here and point more to there, or to move stuff from the project repo into here, or whether some degree of duplication is OK.

Some duplication is okay, but I guess if the website allows us a much better rendering things will naturally move over there

**Comment by Elchi3** [comment]
Ah okay, but when I'm on https://deploy-preview-2--inspiring-snyder-9401af.netlify.app/ (link from the PR preview), and I click on "About us", it brings me to https://inspiring-snyder-9401af.netlify.app/about which is 404. It should have linked to https://deploy-preview-2--inspiring-snyder-9401af.netlify.app/about. Is this something you can control?

File: config.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ languages:
     en:
         languageName: "English"
         weight: 1
+        menu:
+            main:
+                - name: About us
+                  url: about
```
Comment (Elchi3) [inline config.yml:12]: Maybe?

```suggestion
                  url: /about
```

Comment (wbamberg) [reply config.yml]: let's see :)

**Comment by wbamberg** [comment]
> Ah okay, but when I'm on https://deploy-preview-2--inspiring-snyder-9401af.netlify.app/ (link from the PR preview), and I click on "About us", it brings me to https://inspiring-snyder-9401af.netlify.app/about which is 404. It should have linked to https://deploy-preview-2--inspiring-snyder-9401af.netlify.app/about. Is this something you can control?

I think I have fixed this - it seems like you have to tell netlify to use a different base URL for preview builds.

**Review by Elchi3 — APPROVED** [review]
Great work! 👍

**Pull request merged**